apps/web/modules/ee/whitelabel/email-customization/lib/organization.ts

```
@@ -0,0 +1,111 @@
+import { Prisma } from "@prisma/client";
+import { prisma } from "@formbricks/database";
+import { organizationCache } from "@formbricks/lib/organization/cache";
+import { projectCache } from "@formbricks/lib/project/cache";
+import { validateInputs } from "@formbricks/lib/utils/validate";
+import { ZId, ZString } from "@formbricks/types/common";
+import { ResourceNotFoundError } from "@formbricks/types/errors";
+
+export const updateOrganizationEmailLogoUrl = async (
+  organizationId: string,
+  logoUrl: string
+): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)